tool/analyze-and-test-examples.sh

```diff
@@ -45,7 +45,7 @@ function toggleInFileAnalyzerFlags() {
   fi
 
   find $* -name "*.dart" ! -path "**/.*" \
-    -exec perl -i -pe "s{//$mark(ignore(_for_file)?: .*?\b(stable|dev)\b)}{//$toggle\$1}g" {} \;
+    -exec perl -i -pe "s{//$mark(ignore(_for_file)?: .*?\b(stable|beta|dev)\b)}{//$toggle\$1}g" {} \;
```

Comment:
I will be a very happy man when we excise perl from this repo.

Reply:
As will I :)